src/main/kotlin/net/atos/zac/documentcreation/model/DocumentCreationDataAttended.kt

```diff
@@ -5,13 +5,22 @@
 package net.atos.zac.documentcreation.model
 
 import net.atos.client.zgw.zrc.model.Zaak
+import java.time.ZonedDateTime
 
 data class DocumentCreationDataAttended(
-    val taskId: String? = null,
+    val zaak: Zaak,
+
+    val taskId: String?,
```

Comment:
I aassume this is the ID used for handshaking/verification and securing we just talked about; correct? If correct, then I don't understand the optional; if not correct, pls ignore this remark and close directly

Reply:
`taskId` is used when creating document for taak inside zaak. As the same object can be used for creating a document in zaak directly the `taskId` is optional.